// onnxruntime-common@1.20.1/dist/esm/index.js
export { registerBackend } from './backend-impl.js';
export { env } from './env.js';
export { InferenceSession } from './inference-session.js';
export { Tensor } from './tensor.js';
export { TRACE, TRACE_FUNC_BEGIN, TRACE_FUNC_END } from './trace.js';
export { TrainingSession } from './training-session.js';

// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.
/**
 * # ONNX Runtime JavaScript API
 *
 * ONNX Runtime JavaScript API is a unified API for all JavaScript usages, including the following NPM packages:
 *
 * - [onnxruntime-node](https://www.npmjs.com/package/onnxruntime-node)
 * - [onnxruntime-web](https://www.npmjs.com/package/onnxruntime-web)
 * - [onnxruntime-react-native](https://www.npmjs.com/package/onnxruntime-react-native)
 *
 * See also:
 * - [Get Started](https://onnxruntime.ai/docs/get-started/with-javascript/)
 * - [Inference examples](https://github.com/microsoft/onnxruntime-inference-examples/tree/main/js)
 *
 * @packageDocumentation
 */
